app/src/main/java/com/duckduckgo/app/tabs/ui/TabSwitcherAdapter.kt

```diff
@@ -77,6 +77,7 @@ class TabSwitcherAdapter(
     private val list = mutableListOf<TabSwitcherItem>()
     private var isDragging: Boolean = false
     private var layoutType: LayoutType = LayoutType.GRID
+    private var onAnimationTileCloseClickListener: (() -> Unit)? = null
```

Comment:
There's already a `TabSwitcherListener` supplied in the constructor that contains a couple of callbacks, WDYT about moving it there?